packages/tasks/src/tasks/automatic-speech-recognition/about.md

```diff
@@ -48,7 +48,7 @@ await inference.automaticSpeechRecognition({
 });
 ```
 
-For transformers compatible models like Whisper, Wav2Vec2, HuBERT, etc. You can also run inference in Python using transformers as follows:
+This works for transformers-compatible models like Whisper, Wav2Vec2, HuBERT, etc. You can also run inference in Python using transformers as follows:
```

Comment:
Mmm both are unclear to me, maybe this?

```suggestion
For transformers-compatible models like Whisper, Wav2Vec2, and HuBERT, you can also run inference with the library as follows:
```

Reply:
haha, I thought the original intent was to say that the previous api methods work for transformers models. But it looks fine like this imo.

Reply:
ping @Vaibhavs10 for confirmation